src/bootstrap/bootstrap.py

```diff
@@ -463,10 +463,10 @@ def build_triple(self):
             cputype = 'i686'
         elif cputype in {'xscale', 'arm'}:
             cputype = 'arm'
-        elif cputype in {'armv6l', 'armv7l', 'armv8l'}:
+        elif cputype == 'armv6l':
             cputype = 'arm'
             ostype += 'eabihf'
-        elif cputype == 'armv7l':
+        elif cputype in {'armv7l', 'armv8l'}:
```

Comment:
That said, I just tried Termux myself, and I do see `aarch64`.  From their [changelog](https://termux.com/changelog.html#termux-v031-2016-03-22) it seems they started installing 64-bit about a year ago.  Perhaps your installation predates this?

> Start using 64-bit arm packages for capable devices. Up until now Termux used 32-bit arm binaries even on 64-bit devices. This change will affect only new installations - existing users may reinstall the app to get a 64-bit environment (making sure to save relevant material from the home folder first).